mobify/progressive-web-scaffold
# Add the yarn.lock file for anyone using yarn

Adds the `yarn.lock` file for anyone who wants to use `yarn` (https://github.com/yarnpkg/yarn).

 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/298

## Changes
- Created the `yarn.lock` file

## How to test-drive this PR
- Install `yarn` => https://yarnpkg.com/en/docs/install
- Delete your `node_modules` folder
- Run `yarn`
- Run `npm run test:all` and make sure all tests pass
